.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ cassandra_image: &cassandra_image cassandra:3.11.7
 consul_image: &consul_image consul:1.6.0
 moto_image: &moto_image palazzem/moto:1.0.1
 mysql_image: &mysql_image mysql:5.7
-postgres_image: &postgres_image postgres:10.5-alpine
+postgres_image: &postgres_image postgres:11-alpine
```

Comment:
the one the django tests use is in `docker-compose.yml`, we'll need to update it there too.